src/022/p022.coco

```diff
@@ -0,0 +1,7 @@
+parse = open ..> .read() ..> .split(",") ..> sorted
+
+score = .[1:-1] ..> map$(ord ..> (+)$(-64)) ..> reduce$(+)
+
+names = zip$(range(1, 6000)) ..> starmap$(*) ..> reduce$(+)
+
+"p022_names.txt" |> parse |> map$(score) |> names |> print
```

Comment:
Maybe you could wrap this last call in a solve function and call it afterwards. What do you think?

Reply:
Thy wish is my command (once i get home)

Reply:
Also, how can i test these file dependent problems on tio.run?

Reply:
I don't think you can, since there's no way to upload a file to the environment (as far as I know).